experimental/auth-logic/unix_epoch_time_wrapper.go

```diff
@@ -27,7 +27,10 @@ import (
 
 type UnixEpochTime struct{}
 
-func (timeWrapper UnixEpochTime) EmitStatement() UnattributedStatement {
+func (timeWrapper UnixEpochTime) EmitStatement() (UnattributedStatement, error) {
 	epochTime := time.Now().Unix()
-	return UnattributedStatement{Contents: fmt.Sprintf("RealTimeIs(%v).", epochTime)}
+	statement := UnattributedStatement{
```

Comment:
It's fine to inline this in the return (here and elsewhere)

Reply:
fixed